code/datums/diseases/advance/symptoms/symptoms.dm

```diff
@@ -21,12 +21,20 @@ var/global/const/SYMPTOM_ACTIVATION_PROB = 5
 
 /datum/symptom/New()
 	var/list/S = list_symptoms
+
+	//Randomize Symptom Stats
+	stealth += rand(-1,1)

```

Comment:
This is going to randomise the stats everytime the disease is transferred between bottles or when infecting new people.